build.gradle

```diff
@@ -308,10 +310,21 @@ if (IS_WINDOWS && OS_ARCH != "x86" && OS_ARCH != "amd64") {
     fail("Unknown and unsupported build architecture: $OS_ARCH")
 } else if (IS_MAC && OS_ARCH != "x86_64" && OS_ARCH != "aarch64") {
     fail("Unknown and unsupported build architecture: $OS_ARCH")
-} else if (IS_LINUX && OS_ARCH != "i386" && OS_ARCH != "amd64" && OS_ARCH != "aarch64") {
+} else if (IS_LINUX && OS_ARCH != "i386" && OS_ARCH != "amd64" && OS_ARCH != "aarch64" && OS_ARCH != "loongarch64") {
```

Comment:
this could be simplified with 
`... && !IS_LOONGARCH64` which avoids the more error-prone duplicate "loongarch64" string.
(the same applies to replacing the aarch64 check with !IS_AARCH64)

Reply:
Done.